web-client/src/components/MainLayout/index.tsx

```diff
@@ -48,7 +48,7 @@ const MainLayout = observer(({ children }: IMainLayout) => {
                     <NavLinkLogo to="/"><img src={Logo} alt="Logo" /></NavLinkLogo>
                     <LangSelect style={{ height: '35px' }} />
                 </FlexBox>
-                <FlexBox direction="column" padding="16px 8px 8px 8px" scrollY="auto">
+                <FlexBox direction="column" padding="16px 8px 8px 8px" height="100vh" scrollY="auto">
```

Comment:
I see you have a Flexbox above and underneath. Are you sure you want to give 100vh to the inner flexbox?

Usually I have
div flex container column 100vh
  nav as flexitem and maybe also flexcontainer
  content with flex-grow 1
  footer as flexitem and flexcontainer

Reply:
thanks, fixed